### PR TITLE
Admin add collectible2

### PR DESCRIPTION
**Pull request by tomki1**
- go to add collectible and add name, quantity, and jpg of new collectible to add to collectible table
- if successful, should see json response with the newly created collectible_id
- to see the image, go to "http://localhost:3000/addcollectible/imagebytea/25" if the newly created collectible_id = 25
- index.hbs has an example of showing http://localhost:3000/addcollectible/imagebytea/18 so replace 18 with the collectible_id of your newly added collectible to display the image
- I updated the migrations file to include a bytea column in the collectible table. I have not tested if the migration works

### CODE REVIEW

File: routes/addcollectible.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
+const express = require('express');
+const router = express.Router();
+const Collectible = require('../models/collectible.js');
+const Collectible_type = require('../models/collectibletype.js');
```
Comment (notkimroberts) [inline routes/addcollectible.js:4]: @tomki1 This file is not included in your branch.

**Review by notkimroberts — CHANGES_REQUESTED** [review]
@tomki1 Also you need to include the `express-fileupload` and `file-type` packages in `package.json`. You can do this by running `npm install --save express-fileupload file-type`.

File: routes/addcollectible.js (hunk 1 above)
Comment (tomki1) [reply routes/addcollectible.js:4]: @notkimroberts ok, I have now added the express-fileupload and file-type to package.json and added the models/collectibletype.js file

File: sql/migrations/20201015171230_initial.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ exports.up = (knex) => {
         table.bigInteger('collectible_type_id');    // FK
         table.text('name', 128).notNullable();
         table.text('image_url', 128).notNullable();
+        table.bytea('imagebytea');   // add bytea column to table
```
Comment (notkimroberts) [inline sql/migrations/20201015171230_initial.js:9]: @tomki1 This should be `table.binary` to support storing images. Works great by the way!

**Review by notkimroberts — CHANGES_REQUESTED** [review]

File: sql/migrations/20201015171230_initial.js (hunk 2 above)
Comment (tomki1) [reply sql/migrations/20201015171230_initial.js]: @notkimroberts ok, I just updated it to binary. thanks!

**Review by jhhlim — COMMENTED** [review]
code looks good . will test

File: sql/migrations/20201015171230_initial.js (hunk 2 above)
Comment (jhhlim) [reply sql/migrations/20201015171230_initial.js]: table.bytea works , I didn't see a "binary" attribute in PGADMIN

**Review by jhhlim — APPROVED** [review]

**Pull request merged**